### PR TITLE
Initial widget development

### PR DESCRIPTION
**Pull request by leogilmour**
Corresponding documentation

https://github.com/shortlist-digital/documents/tree/master/technical-architecture/content-widgets/image-led-feature

### CODE REVIEW

File: widget-loader-acf.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      'key' => $key . '_advanced_details_tab',
+      'label' => 'Advanced Details',
+      'type' => 'tab',
+      'placement' => $widgetplacement,
+    ),
+  ),
+);
+
+$widget_config["content-types"] = get_option("options_" . $key . "_available_post_types");
+$widget_config["content-sizes"] = array('main'); // main, main-full-bleed, sidebar
```
Comment (thomasdigby) [inline widget-loader-acf.php:53]: Is this comment still relevant? /shrug

Comment (thomasdigby) [reply widget-loader-acf.php:53]: Ah sorry, is this related to the CMS?

Comment (leogilmour) [reply widget-loader-acf.php:53]: this is cms but I must confess I have never taken the time to know what main / main full bleed / sidebar even means 😳

**Comment by leogilmour** [comment]
[WIDGET: Image Led Feature](https://trello.com/c/EFgR8w7u/153-widget-image-led-feature)

File: widget-loader-acf.php (hunk 1 above)
Comment (ecoad) [reply widget-loader-acf.php:53]: It's a feature which we don't currently make use of.

**Comment by ecoad** [comment]
Sorry, can you take a look at this first: https://github.com/shortlist-digital/documents/pull/49

**Comment by leogilmour** [comment]
@ecoad Have done :)

File: widget-loader-acf.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  'display' => 'block',
+  'sub_fields' => array (
+    array (
+      'key' => $key . '_basic_details_tab',
+      'label' => 'Basic Details',
+      'type' => 'tab',
+      'placement' => $widgetplacement,
+    ),
+    array (
+      'key' => $key . '_post',
```
Comment (ecoad) [inline widget-loader-acf.php]: `$key . '_posts',`

File: widget-loader-acf.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      'label' => 'Choose a post to feature',
+      'name' => 'post',
```
Comment (ecoad) [inline widget-loader-acf.php]: 'posts',

File: widget-loader-acf.php [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  'sub_fields' => array (
+    array (
+      'key' => $key . '_basic_details_tab',
+      'label' => 'Basic Details',
+      'type' => 'tab',
+      'placement' => $widgetplacement,
+    ),
+    array (
+      'key' => $key . '_post',
+      'label' => 'Choose a post to feature',
```
Comment (ecoad) [inline widget-loader-acf.php]: "Manually add posts to widget"

File: widget-loader-acf.php [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+      'key' => $key . '_lists',
+      'label' => 'Or choose a list',
+      'name' => 'lists',
+      'type' => 'post_object',
+      'post_type' => array (
+        0 => 'list',
+      ),
+      'return_format' => 'object',
+	  'instructions' => 'If you select a list, the most recent published post in that list will be featured',
+      'ui' => 1,
```
Comment (ecoad) [inline widget-loader-acf.php:40]: Missing:

```
      'multiple' => 1,
```

File: widget-loader-acf.php [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+      'placement' => $widgetplacement,
+    ),
+    array (
+      'key' => $key . '_post',
+      'label' => 'Choose a post to feature',
+      'name' => 'post',
+      'type' => 'post_object',
+      'post_type' => $post_types,
+      'return_format' => 'object',
+      'ui' => 1,
```
Comment (ecoad) [inline widget-loader-acf.php]: Missing:

```
      'multiple' => 1,
```

**Review by ecoad — CHANGES_REQUESTED** [review]
Request changes to have multiple posts and multiple lists

File: widget-loader-acf.php (hunk 5 above)
Comment (leogilmour) [reply widget-loader-acf.php:40]: Oh! Is this feature like a carousel then?

**Comment by leogilmour** [comment]
I _think_ this is good to go now... I've said that a few times though, so ... 😅

**Review by ecoad — APPROVED** [review]

**Comment by ecoad** [comment]
Tagged with v1.0.0

**Pull request merged**